rope_profiling/test.py

```diff
@@ -1,16 +1,22 @@
 # -*- coding: utf-8 -*-
-"""
-Created on Wed Feb 23 15:44:03 2011
+# -----------------------------------------------------------------------------

```

Comment:
Does this directory and the few files within even still need to exist? The code here is Python 2 only (print statements, etc.), its just test code, and rope support itself support has been removed, so not sure why this is still here after the split-plugins and LSP merges...

Reply:
> Does this directory and the few files within even still need to exist?

Let's leave it until @andfoy restores support for it in pyls (right now pyls always uses Jedi for completions).

> and rope support itself support has been removed

It was not removed. Now rope is a inherited by pyls, not by us directly.